SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialStatics.h"
+#include "GeneralProjectSettings.h"
+#include "SpatialConstants.h"
+#include "SpatialGDKSettings.h"
+#include "Utils/ActorGroupManager.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialStatics);
+
+bool USpatialStatics::SpatialNetworkingEnabled()
+{
+	return GetDefault<UGeneralProjectSettings>()->bSpatialNetworking;
+}
+
+bool USpatialStatics::SpatialOffloadingEnabled()
+{
+	return SpatialNetworkingEnabled() && GetDefault<USpatialGDKSettings>()->bEnableOffloading;
+}
+
+bool USpatialStatics::ActorGroupAssociationMatchesWorkerType(const AActor* Actor)
```

Comment:
Is this function meant to work even when `!SpatialNetworkingEnabled() && !SpatialOffloadingEnabled()`? If that's the case, you should consider `WorkerType` being an empty string or whatever the default ends up being for standalone and dedicated server instances (I left a comment on the PR which changes it, too).

Reply:
Yep nice, hadn't considered non-spatialos cases at all here

Reply:
What do you think about renaming it `IsAuthoritativeWorkerType` ?